app/controllers/users_controller.rb

```diff
@@ -22,11 +28,48 @@ def show
     redirect_to signup_path
   end
 
+  def edit
+    @user = User.find_by(id: params[:id])
+  end
+
+  def update
+    @user = User.find_by(id: params[:id])
+    if @user.update(user_params)
+      flash[:success] = t "edit.updated"
+      redirect_to user_path id: @user.id
+    else
+      render "edit"
+    end
+  end
+
+  def destroy
+    user = User.find_by(id: params[:id])
+    if user&.destroy
```

Comment:
```suggestion
    if user.destroy
```
ri thôi nhé, vì có method_load user handle user nil rồi